gapic.sh

```diff
@@ -50,21 +50,21 @@ EOF
 while true; do
   case "$1" in
     -h | --help ) show_help ;;
-    --go_gapic_opt ) GO_GAPIC_OPT="$2"; shift 2 ;;
     --image ) IMAGE="$2"; shift 2 ;;
     -i | --in ) IN="$2"; shift 2 ;;
     -o | --out ) OUT="$2"; shift 2 ;;
+    --options ) PLUGIN_OPTIONS="$2"; shift 2 ;;
     -p | --path ) PROTO_PATH=$2; shift 2 ;;
     -- ) shift; break; ;;
     * ) break ;;
   esac
 done
 
 # Ensure that all required options are set.
-if [ -z "$GO_GAPIC_OPT" ] || [ -z "$IMAGE" ] || [ -z "$IN" ] || [ -z "$OUT" ]; then
+if [ -z "$IMAGE" ] || [ -z "$IN" ] || [ -z "$OUT" ]; then
   cat << EOF
 Required argument missing.
-The --go_gapic_opt, --image, --in, and --out arguments are all required.
+The --image, --in, and --out arguments are all required.
```

Comment:
`--options` is required too right?

Reply:
In the Go generator's case, yes, but the plugin will fail with the proper errors message. We are making this flag on the shortcut script generic and optional so that the script maybe used with any of the gapic images.